src/components/organisms/AssetActions/Pool/TokenList.tsx

```diff
@@ -31,9 +32,9 @@ export default function TokenList({
         <div>
           <Token symbol="OCEAN" balance={ocean} />
           <Token symbol={dtSymbol} balance={dt} />
-          {conversion > 0 && (
+          {conversion.greaterThan(0) && (
             <Conversion
-              price={`${conversion}`}
+              price={`${conversion.toString()}`}
```

Comment:
double string, the template literals already make it a string but to be sure we could remove them and only do:

```suggestion
              price={`${conversion}`}
```

But maybe that `toString()` method does something we actually need so in that case:

```suggestion
              price={conversion.toString()}
```

Reply:
I'd rather go with the recommended usage from Decimal, that is `.toString()`. Maybe it's the same, but why risk it and get some hidden, hard to find bugs.

Reply:
It's the same. We don't need the double string, `.toString()` is enough. In other places where we had the same thing, I removed the template literals.

Reply:
I don't see the mentioned code change, please push